doc/response-tagging.rst

```diff
@@ -14,13 +14,19 @@ Setup
 
     Make sure to :doc:`configure your proxy <proxy-configuration>` for tagging first.
 
-The response tagger is a decorator around a proxy client that implements
-the ``TagCapable`` interface, handling adding tags to responses::
+The response tagger takes an instance of ``TagHeaderFormatterInterface`` as first
+argument which is responsible for providing the header name as well as formatting
+the provided tags into a proper header value. ``HttpCache`` ships with a
+``CommaSeparatedTagHeaderFormatter`` which simply turns an array of tags into a
+comma-separated list but you can provide any tag header formatter you like.
```

Comment:
comma-separated list. If you need a different behaviour, provide your own implementation of the `TagHeaderFormatter` interface.